[2023-02-12T04:05:05.998+0000] {processor.py:153} INFO - Started process (PID=422) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:05:05.999+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:05:06.000+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:05:06.000+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:05:06.028+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:05:06.137+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:05:06.137+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:05:06.177+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:05:06.177+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:05:06.210+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.216 seconds
[2023-02-12T04:05:36.931+0000] {processor.py:153} INFO - Started process (PID=458) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:05:36.932+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:05:36.932+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:05:36.932+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:05:36.956+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:05:36.994+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:05:36.994+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:05:37.032+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:05:37.032+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:05:37.068+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.141 seconds
[2023-02-12T04:06:07.838+0000] {processor.py:153} INFO - Started process (PID=505) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:06:07.840+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:06:07.841+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:06:07.841+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:06:07.858+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:06:07.873+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:06:07.872+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:06:07.911+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:06:07.911+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:06:07.944+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.108 seconds
[2023-02-12T04:06:38.610+0000] {processor.py:153} INFO - Started process (PID=540) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:06:38.621+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:06:38.622+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:06:38.622+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:06:38.644+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:06:38.743+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:06:38.742+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:06:38.782+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:06:38.782+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:06:38.814+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.208 seconds
[2023-02-12T04:07:09.495+0000] {processor.py:153} INFO - Started process (PID=575) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:07:09.496+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:07:09.497+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:07:09.497+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:07:09.530+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:07:09.570+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:07:09.569+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:07:09.621+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:07:09.620+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:07:09.652+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.163 seconds
[2023-02-12T04:07:40.410+0000] {processor.py:153} INFO - Started process (PID=624) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:07:40.422+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:07:40.423+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:07:40.423+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:07:40.445+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:07:40.461+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:07:40.461+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:07:40.508+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:07:40.507+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:07:40.538+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.133 seconds
[2023-02-12T04:08:11.201+0000] {processor.py:153} INFO - Started process (PID=658) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:11.202+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:08:11.203+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:11.203+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:11.226+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:11.311+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:11.310+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:08:11.349+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:11.349+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:08:11.378+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.181 seconds
[2023-02-12T04:08:18.873+0000] {processor.py:153} INFO - Started process (PID=679) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:18.885+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:08:18.887+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:18.886+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:18.911+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:18.929+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:18.929+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:08:18.980+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:18.979+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:08:19.009+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.140 seconds
[2023-02-12T04:08:30.087+0000] {processor.py:153} INFO - Started process (PID=680) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:30.088+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:08:30.089+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:30.089+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:30.109+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:08:30.125+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:30.125+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:08:30.165+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:08:30.165+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:08:30.196+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.111 seconds
[2023-02-12T04:09:00.890+0000] {processor.py:153} INFO - Started process (PID=729) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:00.902+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:09:00.903+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:00.903+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:00.937+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:00.958+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:00.958+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:09:01.003+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:01.002+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:09:01.043+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.158 seconds
[2023-02-12T04:09:14.096+0000] {processor.py:153} INFO - Started process (PID=744) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:14.107+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:09:14.108+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:14.108+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:14.129+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:14.146+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:14.146+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:09:14.188+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:14.188+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:09:14.219+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.126 seconds
[2023-02-12T04:09:33.946+0000] {processor.py:153} INFO - Started process (PID=766) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:33.947+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:09:33.949+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:33.948+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:33.980+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:34.071+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:34.071+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:09:34.110+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:34.109+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:09:34.153+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.213 seconds
[2023-02-12T04:09:39.215+0000] {processor.py:153} INFO - Started process (PID=781) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:39.217+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:09:39.218+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:39.218+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:39.247+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:09:39.263+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:39.263+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:09:39.302+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:09:39.301+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:09:39.328+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.118 seconds
[2023-02-12T04:10:10.001+0000] {processor.py:153} INFO - Started process (PID=816) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:10.012+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:10:10.014+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:10.013+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:10.036+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:10.073+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:10.073+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:10:10.120+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:10.120+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:10:10.153+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.156 seconds
[2023-02-12T04:10:17.215+0000] {processor.py:153} INFO - Started process (PID=831) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:17.216+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:10:17.217+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:17.217+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:17.246+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:17.278+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:17.278+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:10:17.316+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:17.316+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:10:17.345+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.134 seconds
[2023-02-12T04:10:47.970+0000] {processor.py:153} INFO - Started process (PID=865) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:47.971+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:10:47.972+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:47.972+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:47.999+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:10:48.023+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:48.023+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:10:48.075+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:10:48.074+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:10:48.105+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.140 seconds
[2023-02-12T04:11:18.735+0000] {processor.py:153} INFO - Started process (PID=900) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:11:18.736+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:11:18.737+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:11:18.737+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:11:18.760+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:11:18.865+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:11:18.865+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:11:18.907+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:11:18.907+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:11:18.939+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.209 seconds
[2023-02-12T04:11:49.539+0000] {processor.py:153} INFO - Started process (PID=949) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:11:49.541+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:11:49.542+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:11:49.542+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:11:49.563+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:11:49.594+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:11:49.594+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:11:49.640+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:11:49.640+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:11:49.669+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.133 seconds
[2023-02-12T04:12:20.410+0000] {processor.py:153} INFO - Started process (PID=984) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:12:20.411+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:12:20.412+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:12:20.411+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:12:20.432+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:12:20.447+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:12:20.447+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:12:20.486+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:12:20.486+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:12:20.512+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.105 seconds
[2023-02-12T04:12:51.168+0000] {processor.py:153} INFO - Started process (PID=1020) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:12:51.180+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:12:51.181+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:12:51.181+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:12:51.203+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:12:51.296+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:12:51.296+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:12:51.335+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:12:51.335+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:12:51.380+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.215 seconds
[2023-02-12T04:13:21.956+0000] {processor.py:153} INFO - Started process (PID=1069) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:13:21.958+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:13:21.959+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:13:21.958+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:13:21.986+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:13:22.018+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:13:22.018+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:13:22.056+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:13:22.056+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:13:22.092+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.141 seconds
[2023-02-12T04:13:52.768+0000] {processor.py:153} INFO - Started process (PID=1102) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:13:52.781+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:13:52.782+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:13:52.782+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:13:52.817+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:13:52.838+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:13:52.837+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:13:52.881+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:13:52.881+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:13:52.924+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.162 seconds
[2023-02-12T04:14:23.644+0000] {processor.py:153} INFO - Started process (PID=1138) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:14:23.656+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:14:23.657+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:14:23.657+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:14:23.691+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:14:23.790+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:14:23.789+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:14:23.842+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:14:23.841+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:14:23.875+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.236 seconds
[2023-02-12T04:14:54.495+0000] {processor.py:153} INFO - Started process (PID=1187) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:14:54.496+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:14:54.497+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:14:54.497+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:14:54.518+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:14:54.548+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:14:54.547+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:14:54.592+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:14:54.592+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:14:54.622+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.130 seconds
[2023-02-12T04:15:25.333+0000] {processor.py:153} INFO - Started process (PID=1222) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:15:25.345+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:15:25.346+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:15:25.346+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:15:25.377+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:15:25.398+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:15:25.398+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:15:25.441+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:15:25.441+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:15:25.471+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.143 seconds
[2023-02-12T04:15:56.190+0000] {processor.py:153} INFO - Started process (PID=1257) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:15:56.202+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:15:56.203+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:15:56.203+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:15:56.236+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:15:56.329+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:15:56.329+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:15:56.370+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:15:56.370+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:15:56.403+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.218 seconds
[2023-02-12T04:16:26.931+0000] {processor.py:153} INFO - Started process (PID=1307) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:16:26.942+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:16:26.943+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:16:26.943+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:16:26.973+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:16:27.006+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:16:27.005+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:16:27.045+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:16:27.044+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:16:27.083+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.157 seconds
[2023-02-12T04:16:57.681+0000] {processor.py:153} INFO - Started process (PID=1341) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:16:57.693+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:16:57.695+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:16:57.694+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:16:57.717+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:16:57.734+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:16:57.734+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:16:57.779+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:16:57.779+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:16:57.812+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.135 seconds
[2023-02-12T04:17:28.461+0000] {processor.py:153} INFO - Started process (PID=1374) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:28.462+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:17:28.463+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:17:28.463+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:28.487+0000] {processor.py:753} INFO - DAG(s) dict_keys(['dag_with_py_op']) retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:28.570+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:17:28.570+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-12T04:17:28.610+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:17:28.610+0000] {dag.py:3441} INFO - Setting next_dagrun for dag_with_py_op to 2023-02-11T01:57:00+00:00, run_after=2023-02-12T01:57:00+00:00
[2023-02-12T04:17:28.643+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.186 seconds
[2023-02-12T04:17:44.056+0000] {processor.py:153} INFO - Started process (PID=1404) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:44.067+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:17:44.068+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:17:44.068+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:44.077+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:17:44.074+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 6, in <module>
    conn = psycopg2.connect(
NameError: name 'psycopg2' is not defined
[2023-02-12T04:17:44.078+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:44.102+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.050 seconds
[2023-02-12T04:17:53.119+0000] {processor.py:153} INFO - Started process (PID=1410) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:53.131+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:17:53.132+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:17:53.132+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:53.147+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:17:53.144+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:17:53.147+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:17:53.180+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.066 seconds
[2023-02-12T04:18:13.912+0000] {processor.py:153} INFO - Started process (PID=1444) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:13.913+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:18:13.914+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:13.914+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:13.922+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:13.921+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:18:13.922+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:13.950+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.041 seconds
[2023-02-12T04:18:23.988+0000] {processor.py:153} INFO - Started process (PID=1459) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:24.000+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:18:24.001+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:24.001+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:24.014+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:24.012+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:18:24.015+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:24.041+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.058 seconds
[2023-02-12T04:18:35.547+0000] {processor.py:153} INFO - Started process (PID=1475) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:35.559+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:18:35.561+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:35.561+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:35.570+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:35.569+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:18:35.571+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:35.597+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.053 seconds
[2023-02-12T04:18:54.692+0000] {processor.py:153} INFO - Started process (PID=1495) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:54.705+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:18:54.706+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:54.706+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:54.719+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:18:54.716+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:18:54.719+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:18:54.758+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.071 seconds
[2023-02-12T04:19:25.462+0000] {processor.py:153} INFO - Started process (PID=1531) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:25.474+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:19:25.475+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:19:25.475+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:25.487+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:19:25.485+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:19:25.488+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:25.516+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.059 seconds
[2023-02-12T04:19:30.497+0000] {processor.py:153} INFO - Started process (PID=1546) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:30.498+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:19:30.500+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:19:30.499+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:30.511+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:19:30.509+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:19:30.512+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:30.540+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.048 seconds
[2023-02-12T04:19:58.202+0000] {processor.py:153} INFO - Started process (PID=1582) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:58.213+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:19:58.214+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:19:58.214+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:58.223+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:19:58.222+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:19:58.223+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:19:58.251+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.053 seconds
[2023-02-12T04:20:00.235+0000] {processor.py:153} INFO - Started process (PID=1583) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:20:00.236+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:20:00.238+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:20:00.237+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:20:00.250+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:20:00.248+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:20:00.250+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:20:00.280+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.050 seconds
[2023-02-12T04:20:31.042+0000] {processor.py:153} INFO - Started process (PID=1619) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:20:31.054+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:20:31.055+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:20:31.055+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:20:31.065+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:20:31.064+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:20:31.066+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:20:31.092+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.057 seconds
[2023-02-12T04:21:01.799+0000] {processor.py:153} INFO - Started process (PID=1665) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:01.811+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:21:01.812+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:21:01.812+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:01.821+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:21:01.819+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:21:01.822+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:01.845+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.054 seconds
[2023-02-12T04:21:25.521+0000] {processor.py:153} INFO - Started process (PID=1691) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:25.533+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:21:25.534+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:21:25.534+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:25.544+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:21:25.542+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:21:25.544+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:25.570+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.053 seconds
[2023-02-12T04:21:56.263+0000] {processor.py:153} INFO - Started process (PID=1740) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:56.264+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:21:56.264+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:21:56.264+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:56.271+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:21:56.270+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:21:56.271+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:21:56.320+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.061 seconds
[2023-02-12T04:22:09.108+0000] {processor.py:153} INFO - Started process (PID=1761) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:09.109+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:22:09.109+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:09.109+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:09.117+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:09.116+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:22:09.117+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:09.139+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
[2023-02-12T04:22:39.309+0000] {processor.py:153} INFO - Started process (PID=1789) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:39.321+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:22:39.323+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:39.322+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:39.334+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:39.332+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:22:39.334+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:39.362+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.058 seconds
[2023-02-12T04:22:47.901+0000] {processor.py:153} INFO - Started process (PID=1806) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:47.903+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:22:47.904+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:47.904+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:47.916+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:47.915+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:22:47.917+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:47.940+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.043 seconds
[2023-02-12T04:22:53.998+0000] {processor.py:153} INFO - Started process (PID=1812) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:54.004+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:22:54.005+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:54.005+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:54.015+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:54.013+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:22:54.015+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:54.043+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.051 seconds
[2023-02-12T04:22:57.020+0000] {processor.py:153} INFO - Started process (PID=1813) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:57.021+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:22:57.022+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:57.022+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:57.030+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:22:57.029+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:22:57.031+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:22:57.055+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.039 seconds
[2023-02-12T04:23:27.747+0000] {processor.py:153} INFO - Started process (PID=1862) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:27.748+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:23:27.749+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:23:27.749+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:27.755+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:23:27.754+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:23:27.755+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:27.777+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.033 seconds
[2023-02-12T04:23:40.843+0000] {processor.py:153} INFO - Started process (PID=1878) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:40.855+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:23:40.857+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:23:40.857+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:40.869+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:23:40.868+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:23:40.870+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:40.895+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.057 seconds
[2023-02-12T04:23:50.365+0000] {processor.py:153} INFO - Started process (PID=1885) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:50.367+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:23:50.368+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:23:50.368+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:50.380+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:23:50.378+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:23:50.381+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:23:50.407+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.047 seconds
[2023-02-12T04:24:21.089+0000] {processor.py:153} INFO - Started process (PID=1933) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:24:21.100+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:24:21.102+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:24:21.101+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:24:21.109+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:24:21.108+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/connect_pg_local.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/connect_pg_local.py", line 11, in <module>
    port='5433'
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
[2023-02-12T04:24:21.109+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:24:21.136+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.050 seconds
[2023-02-12T04:24:48.793+0000] {processor.py:153} INFO - Started process (PID=1969) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:24:48.805+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:24:48.806+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:24:48.806+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:24:48.808+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:24:48.832+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.043 seconds
[2023-02-12T04:25:19.455+0000] {processor.py:153} INFO - Started process (PID=2004) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:25:19.456+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:25:19.457+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:25:19.457+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:25:19.458+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:25:19.478+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.028 seconds
[2023-02-12T04:25:50.108+0000] {processor.py:153} INFO - Started process (PID=2038) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:25:50.120+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:25:50.121+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:25:50.121+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:25:50.123+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:25:50.138+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.034 seconds
[2023-02-12T04:26:20.845+0000] {processor.py:153} INFO - Started process (PID=2087) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:26:20.856+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:26:20.857+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:26:20.857+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:26:20.859+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:26:20.877+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.038 seconds
[2023-02-12T04:26:51.629+0000] {processor.py:153} INFO - Started process (PID=2122) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:26:51.630+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:26:51.631+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:26:51.631+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:26:51.632+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:26:51.648+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.023 seconds
[2023-02-12T04:27:22.325+0000] {processor.py:153} INFO - Started process (PID=2157) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:27:22.336+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:27:22.338+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:27:22.337+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:27:22.339+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:27:22.362+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.042 seconds
[2023-02-12T04:27:53.069+0000] {processor.py:153} INFO - Started process (PID=2207) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:27:53.070+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:27:53.071+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:27:53.071+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:27:53.072+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:27:53.091+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.025 seconds
[2023-02-12T04:28:23.771+0000] {processor.py:153} INFO - Started process (PID=2243) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:28:23.783+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:28:23.784+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:28:23.783+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:28:23.785+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:28:23.802+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.034 seconds
[2023-02-12T04:28:54.427+0000] {processor.py:153} INFO - Started process (PID=2277) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:28:54.439+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:28:54.440+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:28:54.440+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:28:54.442+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:28:54.462+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.040 seconds
[2023-02-12T04:29:25.141+0000] {processor.py:153} INFO - Started process (PID=2324) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:29:25.142+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:29:25.143+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:29:25.143+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:29:25.144+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:29:25.165+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T04:29:55.896+0000] {processor.py:153} INFO - Started process (PID=2359) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:29:55.897+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:29:55.899+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:29:55.898+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:29:55.901+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:29:55.920+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.029 seconds
[2023-02-12T04:30:26.061+0000] {processor.py:153} INFO - Started process (PID=2396) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:30:26.073+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:30:26.074+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:30:26.074+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:30:26.077+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:30:26.099+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.044 seconds
[2023-02-12T04:30:56.135+0000] {processor.py:153} INFO - Started process (PID=2444) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:30:56.137+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:30:56.138+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:30:56.138+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:30:56.140+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:30:56.158+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T04:31:27.009+0000] {processor.py:153} INFO - Started process (PID=2487) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:31:27.021+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:31:27.022+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:31:27.022+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:31:27.024+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:31:27.048+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.045 seconds
[2023-02-12T04:31:57.802+0000] {processor.py:153} INFO - Started process (PID=2524) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:31:57.814+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:31:57.816+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:31:57.815+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:31:57.817+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:31:57.840+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.044 seconds
[2023-02-12T04:32:28.580+0000] {processor.py:153} INFO - Started process (PID=2576) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:32:28.591+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:32:28.592+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:32:28.592+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:32:28.594+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:32:28.610+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.034 seconds
[2023-02-12T04:32:59.347+0000] {processor.py:153} INFO - Started process (PID=2612) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:32:59.359+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:32:59.360+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:32:59.360+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:32:59.362+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:32:59.383+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.041 seconds
[2023-02-12T04:33:30.067+0000] {processor.py:153} INFO - Started process (PID=2648) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:33:30.079+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:33:30.080+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:33:30.080+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:33:30.082+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:33:30.106+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.045 seconds
[2023-02-12T04:34:00.845+0000] {processor.py:153} INFO - Started process (PID=2698) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:34:00.846+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:34:00.847+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:34:00.846+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:34:00.848+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:34:00.865+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.025 seconds
[2023-02-12T04:34:31.478+0000] {processor.py:153} INFO - Started process (PID=2735) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:34:31.489+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:34:31.490+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:34:31.490+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:34:31.492+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:34:31.528+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.053 seconds
[2023-02-12T04:35:02.149+0000] {processor.py:153} INFO - Started process (PID=2771) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:35:02.151+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:35:02.153+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:35:02.152+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:35:02.154+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:35:02.173+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.029 seconds
[2023-02-12T04:35:32.942+0000] {processor.py:153} INFO - Started process (PID=2821) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:35:32.954+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:35:32.956+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:35:32.956+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:35:32.960+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:35:32.989+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.050 seconds
[2023-02-12T04:36:03.021+0000] {processor.py:153} INFO - Started process (PID=2856) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:36:03.022+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:36:03.024+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:36:03.023+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:36:03.026+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:36:03.047+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.031 seconds
[2023-02-12T04:36:33.835+0000] {processor.py:153} INFO - Started process (PID=2892) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:36:33.847+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:36:33.848+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:36:33.847+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:36:33.850+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:36:33.876+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.047 seconds
[2023-02-12T04:37:04.508+0000] {processor.py:153} INFO - Started process (PID=2927) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:37:04.516+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:37:04.517+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:37:04.517+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:37:04.518+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:37:04.535+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.030 seconds
[2023-02-12T04:37:35.320+0000] {processor.py:153} INFO - Started process (PID=2977) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:37:35.332+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:37:35.333+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:37:35.333+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:37:35.334+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:37:35.353+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.039 seconds
[2023-02-12T04:38:06.135+0000] {processor.py:153} INFO - Started process (PID=3013) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:38:06.137+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:38:06.138+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:38:06.137+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:38:06.140+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:38:06.160+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.030 seconds
[2023-02-12T04:38:36.909+0000] {processor.py:153} INFO - Started process (PID=3057) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:38:36.910+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:38:36.911+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:38:36.911+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:38:36.912+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:38:36.927+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.021 seconds
[2023-02-12T04:39:07.573+0000] {processor.py:153} INFO - Started process (PID=3098) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:39:07.585+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:39:07.586+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:39:07.586+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:39:07.588+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:39:07.614+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.047 seconds
[2023-02-12T04:39:38.371+0000] {processor.py:153} INFO - Started process (PID=3133) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:39:38.383+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:39:38.384+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:39:38.384+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:39:38.386+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:39:38.412+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.046 seconds
[2023-02-12T04:40:09.076+0000] {processor.py:153} INFO - Started process (PID=3183) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:40:09.087+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:40:09.089+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:40:09.088+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:40:09.090+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:40:09.108+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.038 seconds
[2023-02-12T04:40:39.790+0000] {processor.py:153} INFO - Started process (PID=3219) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:40:39.801+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:40:39.802+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:40:39.802+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:40:39.804+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:40:39.828+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.044 seconds
[2023-02-12T04:41:10.595+0000] {processor.py:153} INFO - Started process (PID=3255) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:41:10.607+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:41:10.608+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:41:10.608+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:41:10.611+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:41:10.634+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.044 seconds
[2023-02-12T04:41:41.422+0000] {processor.py:153} INFO - Started process (PID=3305) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:41:41.423+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:41:41.424+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:41:41.424+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:41:41.426+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:41:41.443+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.037 seconds
[2023-02-12T04:42:12.098+0000] {processor.py:153} INFO - Started process (PID=3341) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:42:12.099+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:42:12.100+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:42:12.100+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:42:12.102+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:42:12.120+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T04:42:42.874+0000] {processor.py:153} INFO - Started process (PID=3378) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:42:42.876+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:42:42.877+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:42:42.877+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:42:42.879+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:42:42.901+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T04:43:13.623+0000] {processor.py:153} INFO - Started process (PID=3427) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:43:13.634+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:43:13.635+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:43:13.635+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:43:13.637+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:43:13.682+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.062 seconds
[2023-02-12T04:43:44.395+0000] {processor.py:153} INFO - Started process (PID=3462) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:43:44.407+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:43:44.409+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:43:44.409+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:43:44.412+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:43:44.438+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.049 seconds
[2023-02-12T04:44:15.211+0000] {processor.py:153} INFO - Started process (PID=3497) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:44:15.222+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:44:15.223+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:44:15.223+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:44:15.225+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:44:15.246+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.039 seconds
[2023-02-12T04:44:45.979+0000] {processor.py:153} INFO - Started process (PID=3542) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:44:45.991+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:44:45.993+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:44:45.993+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:44:45.995+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:44:46.017+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.043 seconds
[2023-02-12T04:45:16.726+0000] {processor.py:153} INFO - Started process (PID=3583) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:45:16.728+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:45:16.729+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:45:16.729+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:45:16.731+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:45:16.751+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T04:45:47.298+0000] {processor.py:153} INFO - Started process (PID=3620) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:45:47.310+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:45:47.311+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:45:47.311+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:45:47.313+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:45:47.330+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.036 seconds
[2023-02-12T04:46:17.862+0000] {processor.py:153} INFO - Started process (PID=3669) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:46:17.873+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:46:17.874+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:46:17.874+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:46:17.875+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:46:17.889+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T04:46:48.388+0000] {processor.py:153} INFO - Started process (PID=3705) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:46:48.389+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:46:48.390+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:46:48.390+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:46:48.392+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:46:48.412+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.028 seconds
[2023-02-12T04:47:18.964+0000] {processor.py:153} INFO - Started process (PID=3755) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:47:18.976+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:47:18.977+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:47:18.977+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:47:18.979+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:47:18.998+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.038 seconds
[2023-02-12T04:47:49.511+0000] {processor.py:153} INFO - Started process (PID=3790) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:47:49.522+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:47:49.523+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:47:49.523+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:47:49.525+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:47:49.546+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.040 seconds
[2023-02-12T04:48:20.065+0000] {processor.py:153} INFO - Started process (PID=3840) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:48:20.068+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:48:20.068+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:48:20.068+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:48:20.069+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:48:20.083+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.020 seconds
[2023-02-12T04:48:50.609+0000] {processor.py:153} INFO - Started process (PID=3876) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:48:50.610+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:48:50.610+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:48:50.610+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:48:50.611+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:48:50.623+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.015 seconds
[2023-02-12T04:49:21.101+0000] {processor.py:153} INFO - Started process (PID=3927) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:49:21.110+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:49:21.111+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:49:21.111+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:49:21.112+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:49:21.131+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T04:49:51.586+0000] {processor.py:153} INFO - Started process (PID=3963) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:49:51.597+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:49:51.598+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:49:51.598+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:49:51.599+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:49:51.614+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.030 seconds
[2023-02-12T04:50:22.169+0000] {processor.py:153} INFO - Started process (PID=4013) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:50:22.180+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:50:22.181+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:50:22.181+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:50:22.183+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:50:22.199+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
[2023-02-12T04:50:52.676+0000] {processor.py:153} INFO - Started process (PID=4049) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:50:52.687+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:50:52.688+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:50:52.688+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:50:52.689+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:50:52.704+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.031 seconds
[2023-02-12T04:51:23.246+0000] {processor.py:153} INFO - Started process (PID=4096) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:51:23.257+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:51:23.258+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:51:23.258+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:51:23.259+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:51:23.271+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.029 seconds
[2023-02-12T04:51:53.727+0000] {processor.py:153} INFO - Started process (PID=4132) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:51:53.738+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:51:53.739+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:51:53.739+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:51:53.741+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:51:53.762+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.040 seconds
[2023-02-12T04:52:24.309+0000] {processor.py:153} INFO - Started process (PID=4182) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:52:24.320+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:52:24.321+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:52:24.321+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:52:24.323+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:52:24.344+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.039 seconds
[2023-02-12T04:52:54.850+0000] {processor.py:153} INFO - Started process (PID=4218) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:52:54.851+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:52:54.851+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:52:54.851+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:52:54.853+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:52:54.864+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.017 seconds
[2023-02-12T04:53:21.089+0000] {processor.py:153} INFO - Started process (PID=37) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:53:21.090+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:53:21.091+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:53:21.091+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:53:21.093+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:53:21.112+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.026 seconds
[2023-02-12T04:53:51.993+0000] {processor.py:153} INFO - Started process (PID=88) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:53:51.994+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:53:51.995+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:53:51.994+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:53:51.996+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:53:52.012+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.024 seconds
[2023-02-12T04:54:22.756+0000] {processor.py:153} INFO - Started process (PID=131) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:54:22.757+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:54:22.757+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:54:22.757+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:54:22.758+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:54:22.772+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.017 seconds
[2023-02-12T04:54:52.813+0000] {processor.py:153} INFO - Started process (PID=175) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:54:52.824+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:54:52.825+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:54:52.825+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:54:52.826+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:54:52.837+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.026 seconds
[2023-02-12T04:55:23.405+0000] {processor.py:153} INFO - Started process (PID=217) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:55:23.406+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:55:23.406+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:55:23.406+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:55:23.407+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:55:23.417+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.014 seconds
[2023-02-12T04:55:54.013+0000] {processor.py:153} INFO - Started process (PID=260) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:55:54.024+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:55:54.025+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:55:54.025+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:55:54.026+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:55:54.036+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.025 seconds
[2023-02-12T04:56:24.534+0000] {processor.py:153} INFO - Started process (PID=304) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:56:24.545+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:56:24.546+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:56:24.546+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:56:24.547+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:56:24.562+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.033 seconds
[2023-02-12T04:56:55.024+0000] {processor.py:153} INFO - Started process (PID=345) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:56:55.025+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:56:55.025+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:56:55.025+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:56:55.026+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:56:55.038+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.017 seconds
[2023-02-12T04:57:25.453+0000] {processor.py:153} INFO - Started process (PID=397) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:57:25.465+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:57:25.466+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:57:25.466+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:57:25.467+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:57:25.482+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T04:57:55.961+0000] {processor.py:153} INFO - Started process (PID=433) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:57:55.961+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:57:55.962+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:57:55.962+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:57:55.963+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:57:55.974+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.016 seconds
[2023-02-12T04:58:26.480+0000] {processor.py:153} INFO - Started process (PID=483) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:58:26.481+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:58:26.482+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:58:26.482+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:58:26.483+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:58:26.495+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.017 seconds
[2023-02-12T04:58:57.075+0000] {processor.py:153} INFO - Started process (PID=519) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:58:57.076+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:58:57.078+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:58:57.077+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:58:57.079+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:58:57.093+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.023 seconds
[2023-02-12T04:59:27.158+0000] {processor.py:153} INFO - Started process (PID=568) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:59:27.159+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:59:27.160+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:59:27.160+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:59:27.161+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:59:27.173+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.019 seconds
[2023-02-12T04:59:57.644+0000] {processor.py:153} INFO - Started process (PID=604) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:59:57.655+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T04:59:57.656+0000] {logging_mixin.py:137} INFO - [2023-02-12T04:59:57.656+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:59:57.657+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T04:59:57.669+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T05:00:27.833+0000] {processor.py:153} INFO - Started process (PID=655) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:00:27.844+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:00:27.845+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:00:27.845+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:00:27.847+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:00:27.860+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T05:00:58.381+0000] {processor.py:153} INFO - Started process (PID=690) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:00:58.392+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:00:58.393+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:00:58.393+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:00:58.395+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:00:58.413+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.037 seconds
[2023-02-12T05:01:28.896+0000] {processor.py:153} INFO - Started process (PID=740) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:01:28.907+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:01:28.908+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:01:28.908+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:01:28.909+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:01:28.929+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
[2023-02-12T05:01:59.353+0000] {processor.py:153} INFO - Started process (PID=775) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:01:59.364+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:01:59.365+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:01:59.365+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:01:59.367+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:01:59.384+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.036 seconds
[2023-02-12T05:02:29.968+0000] {processor.py:153} INFO - Started process (PID=825) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:02:29.979+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:02:29.980+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:02:29.980+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:02:29.981+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:02:29.994+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T05:03:00.483+0000] {processor.py:153} INFO - Started process (PID=862) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:03:00.483+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:03:00.484+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:03:00.484+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:03:00.484+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:03:00.498+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.018 seconds
[2023-02-12T05:03:31.028+0000] {processor.py:153} INFO - Started process (PID=912) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:03:31.039+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:03:31.040+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:03:31.040+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:03:31.042+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:03:31.064+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.041 seconds
[2023-02-12T05:04:01.641+0000] {processor.py:153} INFO - Started process (PID=948) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:04:01.652+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:04:01.653+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:04:01.653+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:04:01.654+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:04:01.670+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.033 seconds
[2023-02-12T05:04:32.236+0000] {processor.py:153} INFO - Started process (PID=998) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:04:32.247+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:04:32.249+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:04:32.249+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:04:32.250+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:04:32.266+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
[2023-02-12T05:05:02.851+0000] {processor.py:153} INFO - Started process (PID=1034) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:05:02.862+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:05:02.863+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:05:02.863+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:05:02.864+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:05:02.878+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.028 seconds
[2023-02-12T05:05:33.299+0000] {processor.py:153} INFO - Started process (PID=1084) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:05:33.300+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:05:33.300+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:05:33.300+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:05:33.301+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:05:33.313+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.016 seconds
[2023-02-12T05:06:03.707+0000] {processor.py:153} INFO - Started process (PID=1127) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:06:03.708+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:06:03.709+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:06:03.709+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:06:03.710+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:06:03.724+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.022 seconds
[2023-02-12T05:06:34.114+0000] {processor.py:153} INFO - Started process (PID=1170) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:06:34.125+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:06:34.126+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:06:34.126+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:06:34.128+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:06:34.146+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.036 seconds
[2023-02-12T05:07:04.701+0000] {processor.py:153} INFO - Started process (PID=1221) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:07:04.701+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:07:04.702+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:07:04.702+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:07:04.703+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:07:04.717+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.018 seconds
[2023-02-12T05:07:35.236+0000] {processor.py:153} INFO - Started process (PID=1257) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:07:35.247+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:07:35.248+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:07:35.248+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:07:35.250+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:07:35.269+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.038 seconds
[2023-02-12T05:08:05.776+0000] {processor.py:153} INFO - Started process (PID=1308) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:08:05.787+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:08:05.788+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:08:05.787+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:08:05.789+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:08:05.805+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.031 seconds
[2023-02-12T05:08:36.379+0000] {processor.py:153} INFO - Started process (PID=1344) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:08:36.390+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:08:36.391+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:08:36.391+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:08:36.393+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:08:36.408+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T05:09:06.982+0000] {processor.py:153} INFO - Started process (PID=1395) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:09:06.993+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:09:06.994+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:09:06.994+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:09:06.995+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:09:07.009+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.029 seconds
[2023-02-12T05:09:37.546+0000] {processor.py:153} INFO - Started process (PID=1430) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:09:37.557+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:09:37.558+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:09:37.558+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:09:37.560+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:09:37.577+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
[2023-02-12T05:10:08.143+0000] {processor.py:153} INFO - Started process (PID=1480) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:10:08.143+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:10:08.144+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:10:08.144+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:10:08.145+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:10:08.163+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.022 seconds
[2023-02-12T05:10:38.713+0000] {processor.py:153} INFO - Started process (PID=1517) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:10:38.714+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:10:38.715+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:10:38.714+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:10:38.717+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:10:38.732+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.024 seconds
[2023-02-12T05:14:43.275+0000] {processor.py:153} INFO - Started process (PID=51) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:14:43.286+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:14:43.288+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:14:43.288+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:14:43.289+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:14:43.346+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.084 seconds
[2023-02-12T05:15:13.793+0000] {processor.py:153} INFO - Started process (PID=86) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:15:13.794+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:15:13.794+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:15:13.794+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:15:13.795+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:15:13.807+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.025 seconds
[2023-02-12T05:15:44.549+0000] {processor.py:153} INFO - Started process (PID=123) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:15:44.549+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:15:44.550+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:15:44.550+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:15:44.551+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:15:44.562+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.016 seconds
[2023-02-12T05:16:15.448+0000] {processor.py:153} INFO - Started process (PID=172) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:16:15.460+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:16:15.460+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:16:15.460+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:16:15.461+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:16:15.472+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.028 seconds
[2023-02-12T05:16:46.030+0000] {processor.py:153} INFO - Started process (PID=208) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:16:46.031+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:16:46.033+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:16:46.032+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:16:46.034+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:16:46.050+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.025 seconds
[2023-02-12T05:17:16.729+0000] {processor.py:153} INFO - Started process (PID=258) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:17:16.741+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:17:16.741+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:17:16.741+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:17:16.742+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:17:16.753+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.029 seconds
[2023-02-12T05:17:47.407+0000] {processor.py:153} INFO - Started process (PID=300) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:17:47.418+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:17:47.419+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:17:47.419+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:17:47.420+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:17:47.431+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.026 seconds
[2023-02-12T05:18:18.151+0000] {processor.py:153} INFO - Started process (PID=343) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:18:18.162+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:18:18.163+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:18:18.163+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:18:18.165+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:18:18.179+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.033 seconds
[2023-02-12T05:18:48.826+0000] {processor.py:153} INFO - Started process (PID=385) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:18:48.837+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:18:48.838+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:18:48.838+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:18:48.840+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:18:48.851+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T05:19:19.589+0000] {processor.py:153} INFO - Started process (PID=428) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:19:19.600+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:19:19.601+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:19:19.601+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:19:19.603+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:19:19.617+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T05:19:50.245+0000] {processor.py:153} INFO - Started process (PID=473) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:19:50.256+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:19:50.257+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:19:50.257+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:19:50.258+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:19:50.270+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T05:20:56.852+0000] {processor.py:153} INFO - Started process (PID=36) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:20:56.853+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:20:56.853+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:20:56.853+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:20:56.854+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:20:56.870+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.020 seconds
[2023-02-12T05:22:03.579+0000] {processor.py:153} INFO - Started process (PID=37) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:22:03.582+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:22:03.584+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:22:03.584+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:22:03.585+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:22:03.607+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.030 seconds
[2023-02-12T05:22:34.113+0000] {processor.py:153} INFO - Started process (PID=85) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:22:34.114+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:22:34.115+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:22:34.114+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:22:34.116+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:22:34.130+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.030 seconds
[2023-02-12T05:23:04.302+0000] {processor.py:153} INFO - Started process (PID=123) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:23:04.302+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:23:04.303+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:23:04.303+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:23:04.304+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:23:04.317+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.018 seconds
[2023-02-12T05:23:34.964+0000] {processor.py:153} INFO - Started process (PID=173) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:23:34.966+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:23:34.970+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:23:34.969+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:23:34.971+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:23:34.994+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.034 seconds
[2023-02-12T05:24:05.514+0000] {processor.py:153} INFO - Started process (PID=209) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:24:05.525+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:24:05.526+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:24:05.526+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:24:05.527+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:24:05.538+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.026 seconds
[2023-02-12T05:24:36.049+0000] {processor.py:153} INFO - Started process (PID=261) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:24:36.060+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:24:36.061+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:24:36.061+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:24:36.063+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:24:36.078+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T05:25:06.607+0000] {processor.py:153} INFO - Started process (PID=296) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:25:06.608+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:25:06.608+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:25:06.608+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:25:06.609+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:25:06.624+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.020 seconds
[2023-02-12T05:25:37.267+0000] {processor.py:153} INFO - Started process (PID=346) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:25:37.278+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:25:37.279+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:25:37.279+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:25:37.280+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:25:37.297+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T05:26:07.394+0000] {processor.py:153} INFO - Started process (PID=382) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:26:07.406+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:26:07.407+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:26:07.407+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:26:07.408+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:26:07.424+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
[2023-02-12T05:26:37.982+0000] {processor.py:153} INFO - Started process (PID=433) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:26:37.993+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:26:37.994+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:26:37.994+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:26:37.995+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:26:38.007+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T05:27:08.512+0000] {processor.py:153} INFO - Started process (PID=469) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:27:08.513+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:27:08.513+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:27:08.513+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:27:08.514+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:27:08.527+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.018 seconds
[2023-02-12T05:27:39.071+0000] {processor.py:153} INFO - Started process (PID=519) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:27:39.083+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:27:39.084+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:27:39.084+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:27:39.086+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:27:39.103+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.037 seconds
[2023-02-12T05:28:09.672+0000] {processor.py:153} INFO - Started process (PID=555) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:28:09.673+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:28:09.673+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:28:09.673+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:28:09.674+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:28:09.689+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.020 seconds
[2023-02-12T05:28:40.246+0000] {processor.py:153} INFO - Started process (PID=604) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:28:40.247+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:28:40.248+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:28:40.248+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:28:40.249+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:28:40.262+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.018 seconds
[2023-02-12T05:29:10.825+0000] {processor.py:153} INFO - Started process (PID=641) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:29:10.825+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:29:10.826+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:29:10.825+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:29:10.826+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:29:10.843+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.020 seconds
[2023-02-12T05:29:41.283+0000] {processor.py:153} INFO - Started process (PID=689) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:29:41.294+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:29:41.295+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:29:41.294+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:29:41.295+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:29:41.311+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.031 seconds
[2023-02-12T05:30:11.809+0000] {processor.py:153} INFO - Started process (PID=723) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:30:11.810+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:30:11.811+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:30:11.810+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:30:11.812+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:30:11.825+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.026 seconds
[2023-02-12T05:30:42.371+0000] {processor.py:153} INFO - Started process (PID=774) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:30:42.378+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:30:42.380+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:30:42.379+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:30:42.381+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:30:42.393+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.027 seconds
[2023-02-12T05:31:12.976+0000] {processor.py:153} INFO - Started process (PID=810) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:31:12.988+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:31:12.989+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:31:12.989+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:31:12.991+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:31:13.006+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
[2023-02-12T05:31:43.554+0000] {processor.py:153} INFO - Started process (PID=861) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:31:43.565+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:31:43.566+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:31:43.566+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:31:43.567+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:31:43.581+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.032 seconds
[2023-02-12T05:32:14.170+0000] {processor.py:153} INFO - Started process (PID=899) to work on /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:32:14.182+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/connect_pg_local.py for tasks to queue
[2023-02-12T05:32:14.183+0000] {logging_mixin.py:137} INFO - [2023-02-12T05:32:14.183+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:32:14.185+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/connect_pg_local.py
[2023-02-12T05:32:14.201+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/connect_pg_local.py took 0.035 seconds
